src/pages/index.js

```diff
@@ -38,29 +49,38 @@ const profileDescriptionInput = document.querySelector(
 //Add modal
 const addNewCardButton = document.querySelector(".profile__add-button");
 const addCardModal = document.querySelector("#profile-add-modal");
+const EditAvatarModal = document.querySelector("#profile-avatar-modal");
```

Comment:
`EditAvatarModal`

Every variable in the project should be in `camelCase` (in **lower case** if it's not a class or component)